backend/src/config.ts

```diff
@@ -24,6 +24,22 @@ export const DAPP_DID_URI = loadEnv('DAPP_DID_URI') as Kilt.DidUri
 export const DAPP_NAME = process.env.DAPP_NAME ?? 'Web3-Login-Demo'
 export const JWT_SIGNER_SECRET = loadEnv('JWT_SIGNER_SECRET')
 
+// To determine which CType to use, refer to the CType Hash
+const SOCIAL_KYC_EMAIL_CTYPE = 
+'0x3291bb126e33b4862d421bfaa1d2f272e6cdfc4f96658988fbcffea8914bd9ac' 
+// Required properties for socialKYC  
+export const SOCIAL_KYC_EMAIL_REQUIRED_PORPERTIES = 'Email' 
+// Specify the Social KYC Trusted Attester DID. Credentials issued by this Attester will be accepted.
+export const SOCIAL_KYC_ATTESTER = 
+'did:kilt:4pehddkhEanexVTTzWAtrrfo2R7xPnePpuiJLC7shQU894aY' 
+
+// Configerable Credential types 
+export const CTYPE_HASH = loadEnv('CTYPE_HASH') || SOCIAL_KYC_EMAIL_CTYPE 
```

Comment:
I don't think it's a good idea to default to a very specific CType from a specific attester. If the Ctype is not set, the setup should probably fail. What's the rationale behind this default value?

Reply:
And why are they capitalized?